spec/spec_helper.rb

```diff
@@ -112,19 +121,16 @@
   # aliases for `it`, `describe`, and `context` that include `:focus`
   # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
   config.filter_run_when_matching :focus
-
```

Comment:
Can we undo these white space changes? Looking to have as small of a diff as possible in case we wanna merge back.